modules/shared_utils.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class RobotTypes(Enum):
```

Comment:
Chore: Generally, enums should be singular, eg `RobotType`.

Reply:
Fixed in a9d88c4edc46d440a13b058bb1c073dad906f7e2